examples/demo/App.tsx

```diff
@@ -139,6 +139,18 @@ class App extends React.Component<any, { layoutFile: string | null, model: FlexL
         return null;
     }
 
+    titleFactory = (node: TabNode) => {
```

Comment:
To prevent changes to all the tabs in the demo can you only apply both icon and title to a single tab, say one with name of "Customized Tab" (with id "customizedtab").

Reply:
Thanks for checking! Done!